app/portainer/components/datatables/endpoints-datatable/endpointsDatatableController.js

```diff
@@ -124,20 +124,6 @@ angular.module('portainer.app').controller('EndpointsDatatableController', [
       });
     };
 
-    this.associateOpenAMTAsync = async function (endpoints, deviceId) {
```

Comment:
Is it normal to remove this function?

Reply:
yep, forgot to clarify in the PR description, but this function was meant to be deleted in other PR and I forgot to do so, it's not used